Solutions/Windows Security Events/Analytic Rules/NRT_base64_encoded_pefile.yaml

```diff
@@ -20,9 +20,9 @@ relevantTechniques:
   - T1140
 query: |
  SecurityEvent
-  | where EventID==4688
+  | where EventID == 4688
   | where isnotempty(CommandLine)
-  | where CommandLine contains "TVqQAAMAAAAEAAA"
+  | where CommandLine has "TVqQAAMAAAAEAAA"
```

Comment:
has will not work here, we need to use contains. The B64 encoded PE file will start with this string but will continue after without any deliminating character meaning has will fail to match.

Reply:
Thanks @petebryan for this catch, changes back to "contains".